### ISSUE TITLE
ask for a loan

### ISSUE THREAD

**Issue by Inkimar** [CGI-NRM/SeqDB-documentation#16]
A user should be able to 'loan'.

- [ ]  Information text
- [ ]  email-address

**Comment by Inkimar** [comment]
@thokall , we need to specify this issue more

**Comment by thokall** [comment]
Indeed. As the data is only stored by us, but owned by other governmental organisations the easiest solution at this moment would be an information box with the cgi email and some general information on loan requests. The key is that the users need to able to somehow list all entries that are of interest so the request email to us will contain those names.
